test/common/testvm.py

```diff
@@ -107,6 +107,7 @@ def __init__(self, address=None, image=None, verbose=False, label=None, fetch=Tr
         self.arch = "x86_64"
 
         self.image = image or testinfra.DEFAULT_IMAGE
+        self.atomic_image = self.image in ["rhel-atomic", "fedora-atomic", "continuous-atomic"]
```

Comment:
@dperpeet: indeed, that's a nicer place; I originally was thinking to add a new flag ("is Atomic") to the `DEFAULT_VERIFY` map entries, but that smells like over-engineering. Simply moving the list into that file should do fine IMHO.